website/docs/images/displacy-ent1.html

```diff
@@ -1,36 +1,83 @@
 <div
     class="entities"
-    style="line-height: 2.5; font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'; font-size: 16px"
+    style="
+        line-height: 2.5;
+        font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif,
+            'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+        font-size: 16px;
+    "
```

Comment:
Where do these formatting edits come from?

Reply:
Everything in 94beed2 is just automatically applying Prettier to all files.